sw/device/silicon_creator/lib/drivers/rstmgr_functest.c

```diff
@@ -25,8 +26,8 @@ enum {
 
 bool test_main(void) {
   // Read and clear reset reason.
-  uint32_t reason = rstmgr_reason_get();
-  rstmgr_reason_clear(reason);
+  uint32_t reason = rstmgr_testutils_reason_get();
+  rstmgr_testutils_reason_clear(reason);
```

Comment:
this function does not have a parameter:
```suggestion
  rstmgr_testutils_reason_clear();
```

Reply:
Indeed